src/TestSuite/TestCase.php

```diff
@@ -488,6 +494,10 @@ protected function _assertAttributes($assertions, $string)
                 }
             }
             if ($matches === false) {
+                if ($fullDebug === true) {
+                    debug($string);
+                    debug($regex);
+                }
```

Comment:
I think so. That is the only reason usually you want to have more debug info on how to fix things.